### ISSUE TITLE
Private notes are being sent to Mattermost

### ISSUE THREAD

**Issue by Magicie** [altsol/redmine_mattermost#14]
This plugin also sends private notes to Mattermost, but that is not desired. I only wand ticket updates except private notes. Is there any chance to turn this behaviour off?  Thanks.

**Comment by thanosk** [comment]
@petrhasalik hello and thank you for your feedback. Can you help me reproduce this bug? I'm posting private notes to an open issue for a redmine project and they are not forwarded to mattermost. Perhaps you're following a different path in redmine?

**Comment by Magicie** [comment]
No, that is exactly what I was doing and these notes were forwarded to Mattermost. I had to disable note updates and only use new ticket notifications.

**Comment by thanosk** [comment]
Sorry, you're right. I'll look into it and get back to you.

**Comment by thanosk** [comment]
Hello again, I've posted a patch for this issue, can you please test and confirm it works as intended?

**Comment by bor8** [comment]
Feature request: Can we make this configurable?

**Comment by thanosk** [comment]
Hello, I guess we can turn this into a configuration option, so I'm reopening the issue (but cannot guarrantee when I will find the time to implement it, any PR contribution is always welcome).
